C++/C++.sublime-syntax

```diff
@@ -945,7 +945,7 @@ contexts:
       scope: keyword.operator.c++
     - match: '(?=\b({{control_keywords}}|{{operator_keywords}}|{{casts}}|{{memory_operators}}|{{other_keywords}}|operator)\b)'
       pop: true
-    - match: '(?=\s)'
+    - match: '(?=\s+)'
```

Comment:
Why adding `+`? It's a lookahead, which wouldn't consume it anyway.

Reply:
Indeed, I missed that it is look ahead and does not make any sense. What do you think if we ate here all the whitespace symbols by a normal regexp?